openlibrary/core/lending.py

```diff
@@ -436,8 +421,12 @@ def return_loan(self):
 
     def delete(self):
         loan = dict(self, returned_at=time.time())
+        user_key = self['user']
+        account = OpenLibraryAccount.get(key=user_key)
         if self.get("stored_at") == 'ia':
-            ia_lending_api.delete_loan(self['ocaid'], userkey2userid(self['user']))
+            ia_lending_api.delete_loan(self['ocaid'], userkey2userid(user_key))
+            if account.itemname:
+                ia_lending_api.delete_loan(self['ocaid'], account.itemname)
```

Comment:
How does this invocation of `ia_lending_api.delete_loan()` differ from the one 2 lines earlier? Okay for them both to run?